src/main/scala/com/coralogix/sbtprotodep/package.scala

```diff
@@ -2,5 +2,5 @@ package com.coralogix
 
 package object sbtprotodep {
   lazy val protofetchVersion = "v0.1.0"
```

Comment:
Can be a just-released `0.1.1` now.
```suggestion
  lazy val protofetchVersion = "v0.1.1"
```

Reply:
done